include/exadg/postprocessor/kinetic_energy_spectrum.cpp

```diff
@@ -204,12 +205,16 @@ class DealSpectrumWrapper
 
       // ... permute
       timer.start("Permutation");
-      ArrayView<double> dst(
-        fftw.u_real,
-        s.dim * pow_(static_cast<types::global_dof_index>(s.cells * s.points_dst), s.dim) * 2);
+      ArrayView<double>       dst(fftw.u_real,
+                            s.dim *
+                              dealii::Utilities::pow(static_cast<types::global_dof_index>(
+                                                       s.cells * s.points_dst),
+                                                     s.dim) *
+                              2);
```

Comment:
Is there a way to refactor this code so that the alignment is not that ugly? A first try would be to introduce an empty line between the variables `dst` and `src_` so that there is more space left for the `dst` variable. Or, why not remove the namespace dealii from this file to make the expressions shorter?

Reply:
I introduced a separate variable `size`, see c6b0cb09